common/src/main/java/org/arpha/dto/media/request/FileUploadRequest.java

```diff
@@ -0,0 +1,16 @@
+package org.arpha.dto.media.request;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import org.arpha.dto.media.enums.UploadType;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+public class FileUploadRequest {
+
+    private long targetId;
+    private UploadType targetType;
+
+}
```

Comment:
Consider extending the FileUploadRequest with file metadata like size, mime type etc.